### PR TITLE
chore: clean-up stitched marketing collections

### PR DESCRIPTION
**Pull request by egdbear**
This PR cleans up the environment variables used for unstitched Marketing Collections. It also adds a couple of tests to verify the expected resolved data for the types where we resolve the marketing collections. 

A remaining task is to unset the **environment variables** once this is deployed.
```
hokusai [staging|production] env unset USE_UNSTITCHED_MARKETING_COLLECTION_SCHEMA
hokusai [staging|production]  
```

### CODE REVIEW

**Comment by damassi** [comment]
Thanks for the quick follow-up here with removing this code 🙏

**Review by joeyAghion — APPROVED** [review]
Could you comment with the exact migration? Is it just `hokusai [staging|production] env unset USE_UNSTITCHED_MARKETING_COLLECTION_SCHEMA`?

**Comment by egdbear** [comment]
> Could you comment with the exact migration? Is it just `hokusai [staging|production] env unset USE_UNSTITCHED_MARKETING_COLLECTION_SCHEMA`?

correct! 

```sh
hokusai [staging|production] env unset USE_UNSTITCHED_MARKETING_COLLECTION_SCHEMA
hokusai [staging|production] refresh  
``

File: src/lib/stitching/gravity/schema.ts [hunk 1]
```diff
@@ -54,13 +44,11 @@ export const executableGravitySchema = () => {
     "User",
     "UsersConnection",
     "UsersConnectionEdge",
+    "MarketingCollection",
```
Comment (mzikherman) [inline src/lib/stitching/gravity/schema.ts:47]: Oh! We should clean things up in Gravity as well (unfortunately we don't have the ability to check if a particular field is being requested in DataDog AFAIK)

**Review by mzikherman — DISMISSED** [review]
Really really nice cleanup! 🔥

File: src/lib/stitching/gravity/schema.ts (hunk 1 above)
Comment (egdbear) [reply src/lib/stitching/gravity/schema.ts:47]: that makes sense!

File: _schemaV2.graphql [hunk 2]
```diff
@@ -11924,7 +11931,11 @@ type MarketPriceInsightsEdge {
   node: MarketPriceInsights
 }
 
-type MarketingCollection {
+type MarketingCollection implements Node {
```
Comment (araujobarret) [inline _schemaV2.graphql:11934]: Do you know why MarketingCollection wasn't implementing Node before? I see the interface was added to the type way back

**Review by araujobarret — APPROVED** [review]
🎖️

**Pull request merged**